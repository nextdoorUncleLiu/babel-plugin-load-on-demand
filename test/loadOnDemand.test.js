const babel = require('babel-core')
const loadOnDemand = require('../src/index.js')
it('loadOnDemand', () => {
    const { code } = babel.transform('import { Dialog } from "@mc/mc-components/packages"', {
        plugins: [
            [
                loadOnDemand,
                {
                    "libraryName": "@mc/mc-components/packages"
                }
            ]
        ]
    })
    expect(code).toMatchSnapshot()
})