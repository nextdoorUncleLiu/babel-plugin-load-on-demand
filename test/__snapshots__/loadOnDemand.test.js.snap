// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadOnDemand 1`] = `"import Dialog from \\"@mc/mc-components/packages/Dialog\\";"`;
